packages/graph-editor/package.json

```diff
@@ -72,7 +72,8 @@
   },
   "peerDependencies": {
     "react": "18.2.0",
-    "react-dom": "18.2.0"
+    "react-dom": "18.2.0",
+    "@tokens-studio/sdk": "^0.3.0"
```

Comment:
Looks like we can remove this as its only used in the graph engine ?

Reply:
It's actually used in  ExternalDataContext.tsx